### PR TITLE
Nrf91 board rename

### PR DESCRIPTION
**Pull request by ioannisg**
Renaming of nRF91 DK in the tree (board target names and documentation)

### CODE REVIEW

**Review by anangl — COMMENTED** [review]
One more renaming needed, in [nrf9160dk_nrf9160_partition_conf.dts](https://github.com/zephyrproject-rtos/zephyr/blob/1eb90b96fb8832896c3cb44bc0c36a45b2f65728/boards/arm/nrf9160dk_nrf9160/nrf9160dk_nrf9160_partition_conf.dts#L8).

I think that you should also rename `nrf52840_pca10090` to `nrf9160dk_nrf52840` in the same PR.

File: boards/arm/nrf9160dk_nrf9160/doc/index.rst [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+nRF9160 DK
+##########
 
 Overview
 ********
 
-The nRF9160 PCA10090 is a single-board development kit for evaluation
+The nRF9160 Development Kit (PCA10090) is a single-board development kit
+for evaluation
 and development on the nRF9160 SiP for LTE-M and NB-IoT. The
```
[the remark this answers is not in the document]
Comment (ioannisg) [reply boards/arm/nrf9160dk_nrf9160/doc/index.rst]: ok, will do :)

File: boards/arm/nrf9160dk_nrf9160/Kconfig [hunk 2]
```diff
@@ -1,12 +1,12 @@
-# nRF9160 PCA10090 board configuration
+# nRF9160 DK board configuration
 
 # Copyright (c) 2018 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_NRF52840_GPIO_RESET
 	bool "Use nRF52840 PCA10090 GPIO reset pin"
```
[the remark this answers is not in the document]
Comment (ioannisg) [reply boards/arm/nrf9160dk_nrf9160/Kconfig]: Agreed. I left it because I did not want to touch the nRF52840 on this PR, but i can change it

**Comment by ioannisg** [comment]
> One more renaming needed, in [nrf9160dk_nrf9160_partition_conf.dts](https://github.com/zephyrproject-rtos/zephyr/blob/1eb90b96fb8832896c3cb44bc0c36a45b2f65728/boards/arm/nrf9160dk_nrf9160/nrf9160dk_nrf9160_partition_conf.dts#L8).
> 
> I think that you should also rename `nrf52840_pca10090` to `nrf9160dk_nrf52840` in the same PR.

Will add the commits on top in this PR

**Review by joerchan — APPROVED** [review]

**Comment by ioannisg** [comment]
@anangl pls, take another look

**Comment by anangl** [comment]
@ioannisg Please, remove the file `boards/arm/nrf9160dk_nrf52840/doc/img/nrf52840_pca10056.jpg`. This is not an adequate picture for that board. It seems that this file has been added by mistake in [this commit](https://github.com/zephyrproject-rtos/zephyr/commit/e521ee651f0f5cc9f75501774036ddd48312358e#diff-be100f076811812ed45e16ca44f1a113), as it is not referenced from documentation or any other place.

Also the order of commits should be corrected. The commit that renames nrf52840_pca10090 to nrf9160dk_nrf52840 should be placed before the one that adds the corresponding deprecation warning.

**Comment by ioannisg** [comment]
> @ioannisg Please, remove the file `boards/arm/nrf9160dk_nrf52840/doc/img/nrf52840_pca10056.jpg`. This is not an adequate picture for that board. It seems that this file has been added by mistake in [this commit](https://github.com/zephyrproject-rtos/zephyr/commit/e521ee651f0f5cc9f75501774036ddd48312358e#diff-be100f076811812ed45e16ca44f1a113), as it is not referenced from documentation or any other place.
> 
> Also the order of commits should be corrected. The commit that renames nrf52840_pca10090 to nrf9160dk_nrf52840 should be placed before the one that adds the corresponding deprecation warning.

Yeah, I was thinking the same about this picture when doing the renaming work. Will add a commit that removes it.

**Comment by ioannisg** [comment]
@anangl comments addressed

**Review by anangl — APPROVED** [review]

**Review by lemrey — APPROVED** [review]

**Pull request merged**